src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1138,3 +1163,69 @@ export const getStatus = (response) => {
     ? response.errors[0].status
     : 'unknown';
 };
+
+// Series of utility functions to sort claims and appeals by last updated date
+/**
+ * 
+ * @param {Object} appeal
+ * @returns {string}
+ */
+const getAppealDate = (appeal) => {
+  const { events } = appeal.attributes;
+  const dateString = (events && events.length)
+    ? events[events.length - 1].date
```

Comment:
@cmgiven Are the events in `appeal.attributes.events` always sorted by date (where the last item in the array is the most recent)?